src/extensions/extension-loader.ts

```diff
@@ -337,7 +337,11 @@ export class ExtensionLoader extends Singleton {
     try {
       return __non_webpack_require__(extAbsolutePath).default;
     } catch (error) {
-      logger.error(`${logModule}: can't load extension main at ${extAbsolutePath}: ${error}`, { extension, error });
+      if (ipcRenderer) {
```

Comment:
This also looks like unrelated change?

Reply:
Sure I can revert these, though this error logging in particular I changed because I needed it while debugging why the extensions weren't being added.